src/libs/konnectors/cleanKonnectorsOnBoot.ts

```diff
@@ -38,6 +47,14 @@ export const cleanKonnectorsOnBoot = async (
     store.dispatch(setCurrentRunningKonnector())
   }
 
+  if (runningKonnectorJobId !== undefined) {
+    log.warn(
+      `Stopping the job ${runningKonnectorJobId} as context deadline exceeded`
```

Comment:
Should we use something like `as the connector was not cleaned on previous app session` instead of `as context deadline exceeded`? In my point of view we are not really in a "timeout" scenario here.